src/lib.rs

```diff
@@ -159,33 +159,33 @@ impl LepTess {
     /// lt.set_image("./tests/di.png");
     /// println!("{}", lt.get_utf8_text().unwrap());
     /// ```
-    pub fn get_utf8_text(&self) -> Result<String, std::str::Utf8Error> {
+    pub fn get_utf8_text(&mut self) -> Result<String, std::str::Utf8Error> {
```

Comment:
curious what's the reason for these mut reference changes, is it mutating internal states now?

Reply:
They all have the potential to call `Recognize` https://tesseract-ocr.github.io/tessapi/5.x/a00056_source.html#l01350

https://tesseract-ocr.github.io/tessapi/5.x/a02438.html#a0e4065c20b142d69a2324ee0c74ae0b0

My understanding of `Recognize` is it mutates the internal state of the `tesseract::TessBaseAPI` object.

So because they might call a `&mut self` function, they themselves have to be `&mut self`.

Reply:
Ha, got it, that makes sense, good catch :+1: